src/main/java/application/services/DefaultMenuService.java

```diff
@@ -0,0 +1,66 @@
+package application.services;
+
+import application.exceptions.NothingSelectedException;
+import application.model.Client;
+import application.model.Dish;
+import application.model.Order;
+import application.repositories.DishesRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component("service")
+public class DefaultMenuService implements MenuService {
+
+    private DishesRepository repository;
+
+
+    @Autowired
+    public DefaultMenuService(DishesRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public List<Dish> getMenu() {
+        List<Dish> dishList = repository.findAll();
+        return dishList;
+    }
+
+    @Override
+    public List<Dish> getSortedByPriceMenu() {
+        return new ArrayList<>();
+    }
+
+
+    @Override
+    public Dish findByName(String name) {
+        return repository.findByDishNameIgnoreCase(name);
+
+    }
+
+    public List<Dish> performSortedSearch(String sorter) {
+        if ("price".equals(sorter)) {
+            return repository.findAllByOrderByPriceAsc();
+        }
+        if ("description".equals(sorter)) {
+            return repository.findAllByOrderByDishNameAsc();
+        }
+        return new ArrayList<>();
+    }
+
+    public Order getOrderInformation(HttpServletRequest requestWithOrderInformation, Client client) {
```

Comment:
HttpServletRequest should stay in controller, service should have clean api with types parameters